Wiki update for 10380 (display names for planets and systems) (#36)
endless-sky/endless-sky-wiki: wiki/MapData.md

system <name>
```

The true name of a system must be unique. Use this name to internally reference systems in missions, events, etc.

```html
"display name" <name>
```

> Added in **v. 0.10.11**, this name is displayed to the player. If not defined, it defaults to the true name of the system. Multiple systems can share the same display name.

Reviewer: ```suggestion
Since **v. 0.10.11**: Specifies the name displayed to the player. If not defined, it defaults to the true name of the system. Multiple systems can share the same display name.
```

system <name>
```

The true name of a system must be unique. Use this name to internally reference systems in missions, events, etc.

```html
"display name" <name>
```

> Added in **v. 0.10.9**, this name is displayed to the player. If not defined, it defaults to the true name of the system. Multiple systems can share the same display name.

Author: ```suggestion
Added in **v. 0.10.11**, this name is displayed to the player. If not defined, it defaults to the true name of the system. Multiple systems can share the same display name.
```

**Since v. 0.10.0**: Create a `wormhole` node and then assign it to the planet using `wormhole <name>` (see below).

```html
"display name" <name>
```

> Added in **v. 0.10.11**, this name is displayed to the player. If not defined, it defaults to the true name of the planet. Multiple planets can share the same display name.

Reviewer: ```suggestion
Since **v. 0.10.11**: Specifies the name displayed to the player. If not defined, it defaults to the true name of the planet. Multiple planets can share the same display name.
```

**Since v. 0.10.0**: Create a `wormhole` node and then assign it to the planet using `wormhole <name>` (see below).

```html
"display name" <name>
```

> Added in **v. 0.10.9**, this name is displayed to the player. If not defined, it defaults to the true name of the planet. Multiple planets can share the same display name.

Author: ```suggestion
Added in **v. 0.10.11**, this name is displayed to the player. If not defined, it defaults to the true name of the planet. Multiple planets can share the same display name.
```
